[21.05] gitlab: 13.12.0 -> 13.12.2

petabyteboy opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

risicle commented:
We tend to let master's fix get merged first so we can do a `cherry-pick -xe` for the backport.

petabyteboy commented:
The master PR is open as well (#125271) , I had to deploy this *security* release to some production systems running the release branch anyways, but I can force-push a cherry-picked version once the master commit is merged.

Commit:
gitlab: 13.12.0 -> 13.12.2

https://about.gitlab.com/releases/2021/06/01/security-release-gitlab-13-12-2-released/
Backport of #125271
(cherry picked from commit 2a1c29ef4bacac06f9b677931027bf053952618c)

risicle reviewed (approved):
`nixpkgs-review` inc `nixosTests.gitlab` passes for me, nixos x86_64
